src/utils/file_upload.rs

```diff
@@ -328,12 +301,12 @@ fn upload_files_chunked(
                 context.org,
                 vec![context.project.unwrap().to_string()],
                 checksum,
-                &checksums,
+                chunks,
                 context.release,
                 context.dist,
             )?
         } else {
-            api.assemble_release_artifacts(context.org, context.release()?, checksum, &checksums)?
+            api.assemble_release_artifacts(context.org, context.release()?, checksum, chunks)?
```

Comment:
Why are you calling this for polling? Shouldn't you use a `assemble_artifact_bundle` based on the boolean?